controllers/userController.js

```diff
@@ -0,0 +1,16 @@
+import { userService } from '../services';
+
+const signUp = async (req, res) => {
+  try {
+    const { name, email, user_account, phone_number, password } = req.body;
+
```

Comment:
```suggestion
  try {
    const { name, email, userAccount, phoneNumber, password } = req.body;

```
request body에 담겨오는 키값에는 사실 정확한 컨벤션은 없습니다 :)
프론트엔드랑 통신할 때만 잘 되면 되니까요, 하지만 프론트엔드/백엔드 모두 자바스크립트를 사용하고 있기 때문에
컨벤션은 camelCase로 통일해주면 더 깔끔할 것 같습니다 👍 

파일 내부에 SQL문을 제외한 다른 변수들 카멜케이스로 통일해주세요~